Library/Formula/fontforge.rb

```diff
@@ -33,7 +33,10 @@ class Fontforge < Formula
   depends_on "giflib" => :optional
   depends_on "libspiro" => :optional
   depends_on :x11 => :optional
-  depends_on :python if MacOS.version <= :snow_leopard
+
+  # Segfaults on import without Homebrew's Python
+  # https://github.com/fontforge/fontforge/issues/2353
+  depends_on :python
```

Comment:
Welp. I tore out giant chunks of their detection code in `configure` with no luck whatsoever, I still got insta-segfault every time I tried to `import fontforge` with any Python that wasn't `/usr/bin/python`.